csmakefile

```diff
@@ -43,7 +43,7 @@ manufacturer=Cardinal Peak Technologies
 #   in 16.04's qemu-utils is new enough to handle desired formats
 depends=python (>= 2.6), csmake, qemu-img | qemu-utils, vbox-img
 #depends=python (>= 2.6), csmake, qemu-img, vbox-img
-recommends=python-coverage, chrpath
```

Comment:
good catch - this was taken from packaging and doesn't make sense here.  util-linux and gdisk, otoh, with gpt support now enabled, does make sense.....we should probably include grub EFI parts as well as suggested?